src/State/StateTrait.php

```diff
@@ -102,15 +100,14 @@ public function removeParameter(string $id): static
      *
      * @return mixed The value of the parameter.
      */
-    public function getParameter(string $id)
+    public function getParameter(string $id, mixed ...$default): mixed
     {
         if (isset($this->parameters[$id])) {
             return $this->parameters[$id];
         }
 
-        $args = func_get_args();
-        if (array_key_exists(1, $args)) {
-            return $args[1];
+        if (!empty($default)) {
+            return reset($default);
         }
```

Comment:
Why specifying variadic if it returns first element only?

Reply:
Variadic allow to check when user not pass value. See tests: https://github.com/yiisoft/view/blob/master/tests/ViewTest.php#L363-L388